lib/nylas/collection.rb

```diff
@@ -127,9 +127,13 @@ def resources_path
     end
 
     def find_model(id)
-      instance = model.from_hash({ id: id }, api: api)
-      instance.reload
-      instance
+      response = api.execute(
+        **to_be_executed.merge(
```

Comment:
I believe the double splat and `.merge` might be redundant here, but I could be wrong. Does it work without the splats?

Reference: https://jetrockets.pro/blog/ruby-double-splat-operator-cheatsheet

Reply:
We need to use it here to support with ruby 3.0 https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

Reply:
Oh nice!